### ISSUE TITLE
Project status

### ISSUE THREAD

**Issue by rmoriz** [poise/poise-languages#16]
Hi Noah,

given the "archived" status of https://github.com/poise/poise-python is it safe to say that your poise project ended? I couldn't find a clear expressen (checked your blog and twitter feed) but maybe I've overlooked something.

Thanks

**Comment by coderanger** [comment]
Yes, if there was a critical security issue with my code I might be convinced to come back and fix it but that seems unlikely so I'm pretty done with all Chef things.

**Comment by rmoriz** [comment]
I understand. Thank you for all your efforts over the past 10 years!

**Comment by hdost** [comment]
@coderanger Out of curiosity what have you moved on to?
